start-here-hyperledger.gemspec

```diff
@@ -0,0 +1,22 @@
+
+# coding: utf-8
+
+Gem::Specification.new do |spec|
+  spec.name          = "just-the-docs"
+  spec.version       = "0.3.3"
+  spec.authors       = ["Patrick Marsceill"]
```

Comment:
Do we need to change authors/email information? Filename says `start-here-hyperledger.gemspec`.